pkg/shell/terraform.go

```diff
@@ -342,84 +372,67 @@ func ImportInputs(deploymentGroupDir string, artifactsDir string, expandedBluepr
 	if err != nil {
 		return err
 	}
-	g, err := dc.Config.Group(config.GroupName(filepath.Base(deploymentGroupDir)))
+	bp := dc.Config
+
+	g, err := bp.Group(config.GroupName(filepath.Base(deploymentGroupDir)))
 	if err != nil {
 		return err
 	}
-	outputNamesByGroup, err := config.OutputNamesByGroup(g, dc)
+
+	inputs, err := gatherUpstreamOutputs(deploymentRoot, artifactsDir, g, bp)
 	if err != nil {
 		return err
 	}
-
-	// for each prior group, read all output values and filter for those needed
-	// as input values to this group; merge into a single map
-	allInputValues := make(map[string]cty.Value)
-	for groupName, intergroupOutputNames := range outputNamesByGroup {
-		if len(intergroupOutputNames) == 0 {
-			continue
-		}
-		logging.Info("collecting outputs for group %s from group %s", g.Name, groupName)
-		filepath := outputsFile(artifactsDir, groupName)
-		groupOutputValues, err := modulereader.ReadHclAttributes(filepath)
-		if err != nil {
-			return &TfError{
-				help: fmt.Sprintf("consider running \"ghpc export-outputs %s/%s\"", deploymentRoot, groupName),
-				err:  err,
-			}
-		}
-		intergroupValues := intersectMapKeys(intergroupOutputNames, groupOutputValues)
-		mergeMapsWithoutLoss(allInputValues, intergroupValues)
-	}
-
-	if len(allInputValues) == 0 {
+	if len(inputs) == 0 {
 		return nil
 	}
 
-	var outfile string
+	var outFile string
+	var toImport map[string]cty.Value // input values to be imported
+
 	switch g.Kind() {
 	case config.TerraformKind:
-		outfile = filepath.Join(deploymentGroupDir, fmt.Sprintf("%s_inputs.auto.tfvars", g.Name))
+		outFile = fmt.Sprintf("%s_inputs.auto.tfvars", g.Name)
```

Comment:
It seems to me that the different cases for outFile are not being treated the same anymore. This one doesn't have a directory path prepended to it. Is this intentional?

Reply:
```go
// tf case
outFile = fmt.Sprintf("%s_inputs.auto.tfvars", g.Name)
// packer case
outFile = filepath.Join(modPath, fmt.Sprintf("%s_inputs.auto.pkrvars.hcl", mod.ID))

// after switch
outPath := filepath.Join(deploymentGroupDir, outFile)
```

Both cases end up in `deploymentGroupDir`.